manipulate_wheels.py

```diff
@@ -179,27 +180,45 @@ def main():
                             numpy_req_found = True
                             if args.verbose:
                                 print('Found numpy dependency.')
-                            curr_version_req = curr_req.replace("numpy","").replace("(","").replace(")","").strip()
+                            curr_version_req = curr_req.split(';')[0].replace("numpy","").replace("(","").replace(")","").strip()
                             req_tokens = ["numpy",""]
+                            if len(curr_req.split(';')) == 2:
+                                req_tokens += [';' + curr_req.split(';')[1]]
+
+                            min_numpy_set = False
+                            lt_numpy_set = False
+                            req_versions = []
                             if curr_version_req:
                                 # split version spec-set into separate specs
                                 req_versions = curr_version_req.split(',')
                                 for i, version_spec in enumerate(req_versions):
                                     if '>' in version_spec:
                                         # case: old minimum version is lower than our minimal version
-                                        if version.parse(re.sub(REQ_SEP, '', version_spec)) < version.parse(args.set_min_numpy):
+                                        if args.set_min_numpy and version.parse(re.sub(REQ_SEP, '', version_spec)) < version.parse(args.set_min_numpy):
```

Comment:
Here I'd also replace `version.parse(re.sub(REQ_SEP, '', version_spec))` with `parsed_version` or something like that to avoid doing that multiple times.

Reply:
should be better now